samcli/lib/build/app_builder.py

```diff
@@ -51,6 +51,12 @@ def _get_workflow_config(runtime):
             dependency_manager="pip",
             application_framework=None,
             manifest_name="requirements.txt")
+    elif runtime.startswith("nodejs"):
```

Comment:
Noo, lets not do if else's again, this might become like our entrypoint file. Given that there may be more runtimes that we support building.

Reply:
What is your suggestion then?

Reply:
Suggestion is to be encapsulate in a dictionary, and invoke corresponding actions based on a match. But I will not block on this. we can address this separately, I'll create an issue.